pkg/oci/load_balancer_spec.go

```diff
@@ -109,11 +109,24 @@ func NewLBSpec(svc *v1.Service, nodes []*v1.Node, defaultSubnets []string, sslCf
 	// NOTE: These will be overridden for existing load balancers as load
 	// balancer subnets cannot be modified.
 	subnets := defaultSubnets
+	var subnetOverrides [2]string
```

Comment:
Since we're just making sure that there are two subnets defined either in defaults OR in annotations we could simplify this all to just 

```go
subnets := defaultSubnets
if s, ok := svc.Annotations[ServiceAnnotationLoadBalancerSubnet1]; ok {
    subnets[0] = s
}
if s, ok := svc.Annotations[ServiceAnnotationLoadBalancerSubnet2]; ok {
    subnets[1] = s
}

if subnets[0] == "" || subnets[1] == "" {
    return nil, errors.Errorf("annotations for both subnets must be specified")
}
```